loader/goroot.go

```diff
@@ -225,9 +225,22 @@ func needsSyscallPackage(buildTags []string) bool {
 	return false
 }
 
-// The boolean indicates whether to merge the subdirs. True means merge, false
-// means use the TinyGo version.
-func pathsToOverride(goMinor int, needsSyscallPackage bool) map[string]bool {
+// linuxNetworking returns whether the unmodified go linux net stack should be used
+// until the full rework of the net package is done.
+func linuxNetworking(buildTags []string) bool {
+	for _, tag := range buildTags {
+		if tag == "linux" {
+			return true
```

Comment:
Unfortunately, `linux` is also set on baremetal systems (because we have to pretend to be _some_ OS). So this isn't quite right.

I think checking for the equivalent of `linux && !baremetal && !nintendoswitch && !tinygo.wasm` would work here.